bitnami/mongodb/values-production.yaml

```diff
@@ -411,7 +504,7 @@ persistence:
 service:
   ## Service type
   ##
-  type: ClusterIP
```

Comment:
Please don't change the default values, this applies to all the changes in the values.yaml